ibis/backends/dask/execution/arrays.py

```diff
@@ -0,0 +1,59 @@
+import itertools
+import operator
+
+import dask.dataframe as dd
+from dask.dataframe.groupby import SeriesGroupBy
+
+import ibis.expr.operations as ops
+from ibis.backends.pandas.execution.arrays import execute_node
+
+
+@execute_node.register(ops.ArrayLength, dd.Series)
+def execute_array_length(op, data, **kwargs):
+    return data.apply(len)
+
+
+@execute_node.register(ops.ArraySlice, dd.Series, int, (int, type(None)))
+def execute_array_slice(op, data, start, stop, **kwargs):
+    return data.apply(operator.itemgetter(slice(start, stop)))
+
+
+@execute_node.register(ops.ArrayIndex, dd.Series, int)
+def execute_array_index(op, data, index, **kwargs):
+    return data.apply(
+        lambda array, index=index: (
```

Comment:
why do we need the "index" param here as supposed to

```
data.apply(lambda array: ....)
```
?

Reply:
Oh, is this to make sure the value of index is capture at the lambda definition time rather rather than invocation time?